City-of-Pori/pori-kada
# PORI-28 Changed the media-query of the navigation to 1024px

In order to make the navigation trigger later changed the media query for mobile navigation. Enalbed also other pori features overrides.

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Make sure all the Pori features are enabled.
3. Make sure the mobile navigation changes to desktop at 1024 px wide screen.
